packages/verification-vue-component/src/components/Verification/item/EventDetails.vue

```diff
@@ -19,14 +26,15 @@
 <script>
 import { mdiCheck } from '@mdi/js'
 import i18nWrapperMixin from '../../../mixins/i18n-wrapper'
-import MDIcon from '../../MDIcon.vue'
+// Register vue tooltip
+import Vue from 'vue'
+import { VTooltip } from 'v-tooltip'
+
+Vue.directive('tooltip', VTooltip)
```

Comment:
I'm not sure if that's necessary, I guess so, but in the VerificationDemo.vue we haven't done it, but I'm not sure if it still works there as this component is not used at the moment. But at least I think it would make sense to do that as high as possible, so in the CertifactionVerification.vue.

Reply:
Yeah, I had to import it otherwise it would throw a `Component not found` error in the console.

Also, I registered it in the component so that it won't get imported into other components where it's not being used

Reply:
Ok makes sense, for some reason I don't like it when Vue is imported multiple times, but it's probably fine.

Reply:
Yeah, true. I just pushed a change. I used the directives object instead and removed the vue import